app/views/content/blog/what-to-expect-on-your-teacher-training.md

```diff
@@ -23,7 +23,7 @@ closing_paragraph: enriching-the-lives-of-young-people
 
 $seminar_room$
 
-_Jan Ashbridge is a Principal Lecturer with responsibility for primary PGCE at the University of Cumbria. She has particular expertise in early literacy and systematic synthetic phonics._
+Jan Ashbridge is a Principal Lecturer with responsibility for primary PGCE at the University of Cumbria. She has particular expertise in early literacy and systematic synthetic phonics.
```

Comment:
Suggest moving this paragraph to end just before the footer, with an h2 of 'About the author'